data/.schema.yaml

```diff
@@ -33,14 +33,37 @@ $defs:
         description: Specimen type.
         type: string
         enum:
-        - stool
-        - sputum
-        - throat_swab
+          - stool
+          - sputum
+          - throat_swab
+      biomarker:
+        description: Biomarker being measured.
+        type: string
+        enum:
+          - SARS-CoV-2
+          - mtDNA
+          - PMMoV
+      unit:
+        description: Units in which measurements are reported.
+        type: string
+        enum:
+          - gc/dry gram
+          - gc/mL
+          - gc/swab
+          - gc/wet gram
+      reference_event:
+        description: Event used as reference for `time` fields of measurements.
+        type: string
+        enum:
+          - symptom onset
```

Comment:
Can you add an option as "confirmation date"?

Reply:
Yup, should be done in the revised commit.